variables.tf

```diff
@@ -0,0 +1,92 @@
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `eg` or `cp`)"
+}
+
+variable "stage" {
+  type        = "string"
+  description = "Stage (e.g. `prod`, `dev`, `staging`)"
+}
+
+variable "name" {
+  type        = "string"
+  description = "Name  (e.g. `app` or `db`)"
+}
+
+variable "delimiter" {
+  type        = "string"
+  default     = "-"
+  description = "Delimiter to be used between `namespace`, `stage`, `name` and `attributes`"
+}
+
+variable "attributes" {
+  type        = "list"
+  default     = []
+  description = "Additional attributes (e.g. `1`)"
+}
+
+variable "tags" {
+  type        = "map"
+  default     = {}
+  description = "Additional tags (e.g. `{ BusinessUnit = \"XYZ\" }`"
+}
+
+variable "acl" {
+  type        = "string"
+  default     = "log-delivery-write"
+  description = "The canned ACL to apply. We recommend log-delivery-write for compatibility with AWS services"
+}
+
+variable "policy" {
+  type        = "string"
+  default     = ""
+  description = "A valid bucket policy JSON document. Note that if the policy document is not specific enough (but still valid), Terraform may view the policy as constantly changing in a terraform plan. In this case, please make sure you use the verbose/specific version of the policy."
+}
+
+variable "region" {
+  type        = "string"
+  default     = ""
+  description = "If specified, the AWS region this bucket should reside in. Otherwise, the region used by the callee."
+}
+
+variable "force_destroy" {
+  type        = "string"
+  default     = "false"
+  description = "A boolean string that indicates all objects should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable."
+}
+
+variable "versioning_enabled" {
+  type        = "string"
+  default     = "false"
+  description = "A state of versioning. Versioning is a means of keeping multiple variants of an object in the same bucket."
+}
+
+variable "sse_algorithm" {
+  type        = "string"
+  default     = "AES256"
+  description = "The server-side encryption algorithm to use. Valid values are `AES256` and `aws:kms`"
+}
+
+variable "kms_master_key_id" {
+  type        = "string"
+  default     = ""
+  description = "The AWS KMS master key ID used for the SSE-KMS encryption. This can only be used when you set the value of sse_algorithm as aws:kms. The default aws/s3 AWS KMS master key is used if this element is absent while the sse_algorithm is aws:kms"
```

Comment:
```suggestion
  description = "The AWS KMS master key ID used for the `SSE-KMS` encryption. This can only be used when you set the value of `sse_algorithm` as `aws:kms`. The default aws/s3 AWS KMS master key is used if this element is absent while the `sse_algorithm` is `aws:kms`"
```